Source/MediaStorage.swift

```diff
@@ -364,4 +372,84 @@ public class MediaStorage {
+        if accessToken == nil {
+            completionHandler(Dictionary<String, String>(), MediaStorageError(statusCode: nil, message: "wrong usage: use the connect method to get an access token."))
+            return
+        }
+        var isUserKey = false
+        var url : String
+        var userMetaKey = ""
+        if fieldName == nil {
+            completionHandler(Dictionary<String, String>(), MediaStorageError(statusCode: nil, message: "invalid parameter: nil"))
+            return
+        } else if fieldName == metaExif || fieldName == metaGpano || fieldName == metaUser {
+            // GET /media/{id}/meta/exif, /media/{id}/meta/gpano, /media/{id}/meta/user
+            url = "\(mstorageEndpoint)/\(mediaId)\(getMetaPath)/\(fieldName)"
+        } else {
+            userMetaKey = replaceUserMeta(fieldName)
+            if userMetaKey == "" {
+                completionHandler(Dictionary<String, String>(), MediaStorageError(statusCode: nil, message: "invalid parameter: \(fieldName)"))
+                return
+            } else {
+                // GET /media/{id}/meta/user/{key}
+                url = "\(mstorageEndpoint)/\(mediaId)\(getUserMetaPath)/\(userMetaKey)"
+                isUserKey = true
+            }
+        }
+
+        MediaStorageRequest.get(
+            url: url,
+            queryParams: [String: String](),
+            header: [
+                "Authorization" : "Bearer \(accessToken!)"
+            ]
+        ){(data, resp, err) in
+            if err != nil {
+                completionHandler(Dictionary<String, String>(), MediaStorageError(statusCode: nil, message: "request failed: \(err!.code): \(err!.domain)"))
+                return
+            }
+
+            let httpresp = resp as! NSHTTPURLResponse
+            let statusCode = httpresp.statusCode
+            let dataString = NSString(data: data!, encoding: NSUTF8StringEncoding) as! String
+            if !httpresp.isSucceeded() {
+                completionHandler(
+                    Dictionary<String, String>(),
+                    MediaStorageError(statusCode: statusCode, message: "received error: \(dataString)")
+                )
+                return
+            }
+
+            var dataDic = Dictionary<String, String>()
+            if isUserKey {
+                dataDic[userMetaKey] = dataString
+
+            } else  {
+                do {
+                    dataDic = try NSJSONSerialization.JSONObjectWithData(data!, options: NSJSONReadingOptions.AllowFragments) as! [String : String]
+                } catch {
+                    completionHandler(
+                        Dictionary<String, String>(),
+                        MediaStorageError(statusCode: statusCode, message: "invalid response: \(dataString)")
+                    )
+                }
+
+            }
+
+            completionHandler(
+                dataDic,
+                MediaStorageError(statusCode: nil, message: nil)
+            )
+
```

Comment:
Same as this [comment](https://github.com/hishida/media-storage-swift/pull/14/files#r70571589).